samples/tab-user-profile/ts/src/components/profileTab.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import React, { useEffect, useState } from 'react';
+import { app, profile } from "@microsoft/teams-js";
+import validator from 'validator';
+import './Tab.css';
+
+// function for opening user profile card
+function ProfileTab() {
+  const [selectedPeople, setSelectedPeople] = useState("");
+  const [requiredValidation, setRequiredValidation] = useState(false);
+
+  useEffect(() => {
+    app.initialize()
+  })
+
+  // set the input Email value to a state variable
+  const setPeople = (e: any) => {
+    setSelectedPeople(e.target.value)
+  }
+
+  // Checks whether the platform supports Profile Module and shows the Profile card
```

Comment:
// Checks whether the platform supports Profile Module and shows the user profile card.

Reply:
Fixed